src/main.cpp

```diff
@@ -131,7 +124,12 @@ static InkayStatus Inkay_GetStatus() {
 
 static void Inkay_Initialize(bool apply_patches) {
     if (Config::initialized)
-    return;
+        return;
+
+    if (Config::block_initialize) {
+        ShowNotification("Failed to init Inkay. Please restart the console to use Pretendo");
```

Comment:
I think the text should be changed too, to make clear that the issue is not being able to activate the Inkay plugin while from an inactive mode. I also removed the Pretendo part since the plugin may not necessarily have Pretendo mode set:

```suggestion
        ShowNotification("Cannot load Inkay while the system is running. Please restart the console");
```